numba/tests/test_parfors.py

```diff
@@ -1421,9 +1422,14 @@ def test_impl(A):
         # this doesn't fuse due to mixed indices
         self.assertEqual(countParfors(test_impl, (numba.float64[:,:],)), 2)
 
-        def test_impl(A):
-            min_val = np.amin(A)
-            return A - min_val
+        if numpy_version < (1, 25):
+            def test_impl(A):
+                min_val = np.amin(A)
+                return A - min_val
+        else:
+            def test_impl(A):
+                min_val = np.min(A)
+                return A - min_val
```

Comment:
What is the purpose of this change? Is it an artifact of fixing the test prior to discovering that `np.amin` and `np.min` do not alias in 1.25? If so it can probably be reverted?

Reply:
This needs reverting. It's from 1a1c7c1bd5a6cc96cb8e6110480a3bf83d72a5eb and before new overloads were added

Reply:
`np.min` is not available in NumPy 1.22

Reply:
I think `min` is just an alias of `amin` in NumPy 1.22.x:
```pycon
>>> import numpy as np
>>> np.__version__
'1.22.3'
>>> np.min
<function amin at 0xsomeaddress>
```

I'm also of the view that this change needs reverting, `amin` is the function being tested originally and should continue to be tested invariant of how NumPy have implemented it.